src/Compression/CompressionDetectorHelper.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\FileStorage\Compression;
+
+final class CompressionDetectorHelper
+{
+    public static function isGzipped(string $filePath): bool
+    {
+        return in_array(strtolower(pathinfo($filePath, PATHINFO_EXTENSION)), [
```

Comment:
Tohle mi přijde trochu vaření na vodě. Tu kompresi bysme měli detekovat z obsahu souboru přes finfo_file/finfo_open

Reply:
To je copy paste zatím nebylo nic jiného potřeba https://github.com/keboola/connection/blob/cc7761bcae781975e34229c36e48c15e36314840/Package/FileStorage/src/FileHeaderDownloader/Helper.php#L7

Reply:
Fuj to jsem ti tenkrat pustil jo? :D ale ten finfo funguje i na ty chunky, tam se to detekuje ze zacatku souboru. To by nemel byt problem.

Reply:
No už tenkrát to byl copy paste :D Problém je, že abys použil file info tak budeš muset ten soubor stáhnout celý, což docela popírá důvod proč to existuje ten PartialDownloader :/

Reply:
No on finfo funguje na buffer takže to možná nestáhne celé. Ale stejně sem ten Range vyhodil protože na gzip to nefunguje a je potřeba celý soubor :/

Reply:
Prave ze ne, stahnes jen ten chunk a ulozis ho do temp souboru. Overoval jsem si to na testovacim scriptu. Ono to bude detekovat ten typ podle zacatku souboru a to ze ten zip nemas stazeny cely, nebo to ma jinou koncovku atd na to nema vliv.

Reply:
No hodím na to oko, mám enom obavu aby nám to nenadělalo nějaké další problémy, ale tak mělo by to byt cajk.

Reply:
Upraveno udělal sem teda CompressionDetector,který je přímo pro ABS a používá finfo
https://github.com/keboola/php-file-storage-utils/pull/4/commits/1736d7946fc810f6f466a5a5e8e704c0b860c703

Reply:
Super